services/api.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+export const get = async <T>(url: string, params: Record<string, number | string | Date >): Promise<{
```

Comment:
create type for `Record<string, number | string | Date >`

Reply:
done